src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -539,6 +553,39 @@ const formConfig = {
             const requestsRecords = _.get(`disabilities.${index}.view:uploadPrivateRecords`, formData) === 'no';
             return hasRecords && requestsRecords;
           },
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:description': authorizationToDisclose
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {}
+                }
+              }
+            }
+          }
+        },
+        privateMedicalRecordRelease: {
+          title: '',
+          path: 'supporting-evidence/:index/private-medical-records-release',
+          showPagePerItem: true,
+          itemFilter: (item) => _.get('view:selected', item),
+          arrayPath: 'disabilities',
+          // TODO: Re-enable actual depends logic for page once 4142 PDF generation is working through vets-api
+          depends: () => false,
```

Comment:
Might make more sense to comment the whole page out so it doesn't get processed by the library at all. Minor optimization, but it's pretty simple, right?

Reply:
Yeah, having that much commented out seemed sloppy, but assuming this is just a temporary situation, you're probably right. Plus, I could get rid of the extra `depends` code.

Reply:
Actually, commenting out the whole page breaks the test files, and also requires commenting out some imports to satisfy es-lint. I think it's probably an easier change to just keep the extra `depends`, as-is.